docs/src/pages/components/data-grid/filtering/filtering.md

```diff
@@ -93,15 +126,15 @@ Filtering can be run server-side by setting the `filterMode` prop to `server`, a
 />
 ```
 
-Below is very simple demo on how you could achieve server side filtering.
+Below is a very simple demo on how you could achieve server side filtering.
 
 {{"demo": "pages/components/data-grid/filtering/ServerFilterGrid.js", "bg": "inline"}}
 
-### Controlled filtering
+<!-- ## Controlled filtering -->
```

Comment:
I would have personally kept the header up and clarify the description (WIP is about the docs, not the implementation), but the current approach can work too.

Reply:
I want to look into this controlled state and filters this week